crates/ef-testing/src/models/result.rs

```diff
@@ -107,25 +104,25 @@ pub(crate) fn extract_output_and_log_execution_result(
             }
             None
         }
-        TransactionExecutionResult::Err(TransactionExecutionError::ValidateTransactionError(
-            EntryPointExecutionError::VirtualMachineExecutionErrorWithTrace { trace, .. },
-        )) => {
-            // There are specific test cases where validation failed because the sender account has code.
-            // They're caught by EOA validation, and rejected with this specific error message.
-            if trace.contains("EOAs cannot have code") {
-                return None;
-            }
-            let re = regex::Regex::new(
-                r#"Error in the called contract \((0x[0-9a-zA-Z]+)\)[\s\S]*?EntryPointSelector\(StarkFelt\("(0x[0-9a-zA-Z]+)"\)\)"#,
-            ).unwrap();
-            let matches: Vec<_> = re.captures_iter(trace).map(|c| c.extract::<2>()).collect();
-            let last_match = matches.last().cloned().unwrap_or_default();
-            warn!(
-                "Failed to find entrypoint {} for contract {}",
-                last_match.1[1], last_match.1[0]
-            );
-            None
-        }
+        // TransactionExecutionResult::Err(TransactionExecutionError::ValidateTransactionError(
```

Comment:
hmmm can we uncomment this?

Reply:
Just left that here to have your opinion before removing, unfortunately we don't have the trace anymore here: 

https://github.com/starkware-libs/blockifier/blob/bad24f2543c848202f8fe2c833cd4149fef75d9f/crates/blockifier/src/transaction/errors.rs#L101

So that we can't do this logic anymore.

As the branch below the match prints the error, I figured it was enough for this case.

Reply:
removed as discussed